kube-monitoring/charts/Chart.yaml

```diff
@@ -22,4 +22,4 @@ dependencies:
     version: 65.4.0
   - name: kubernetes-operations
     repository: https://cloudoperators.github.io/kubernetes-operations
-    version: 1.1.1
+    version: 1.1.4
```

Comment:
Question: Is there any specific reason to bump the version from `1.1.1` directly to `1.1.4`?

Reply:
Yes, if we move this dashboard directly to `kube-monitoring`, helm will complain about the installation because it is already part of the `kube-operations` package. 

https://github.com/cloudoperators/greenhouse-extensions/actions/runs/11835396250/job/32978079413#step:10:44

Reply:
I mentioned the PR: https://github.com/cloudoperators/kubernetes-operations/pull/14

Reply:
Thanks a lot. My bad.